src/main/java/org/opensearch/ad/transport/handler/MultiEntityResultHandler.java

```diff
@@ -73,7 +73,7 @@ public MultiEntityResultHandler(
      * @param listener callback after flushing
      */
     public void flush(ADResultBulkRequest currentBulkRequest, ActionListener<ADResultBulkResponse> listener) {
-        if (indexUtils.checkIndicesBlocked(clusterService.state(), ClusterBlockLevel.WRITE, this.indexName)) {
+        if (indexUtils.checkIndicesBlocked(sdkClusterService.state(), ClusterBlockLevel.WRITE, this.indexName)) {
```

Comment:
seems you have missed the clusterService name change in the constructor for the `MultiEntityResultHandler` class, curious as to how this passed the build

Reply:
ACk